src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -8316,7 +8316,7 @@ void ObjectMgr::LoadVendors()
 
     std::set<uint32> skip_vendors;
 
-    QueryResult result = WorldDatabase.Query("SELECT entry, item, maxcount, incrtime, ExtendedCost FROM npc_vendor ORDER BY entry, slot ASC");
+    QueryResult result = WorldDatabase.Query("SELECT entry, item, maxcount, incrtime, ExtendedCost FROM npc_vendor ORDER BY entry, slot ASC, item, ExtendedCost");
```

Comment:
the SQL format is wrong, ` ASC` should be at the end

Reply:
lol, fixed

Reply:
> `ASC` should be at the end

The manual for MySQL 8.0 says https://dev.mysql.com/doc/refman/8.0/en/sorting-rows.html
> The DESC keyword applies only to the column name immediately preceding it
> The default sort order is ascending

So it should be immediately after the column the sorting was intended for and the default sorting is ASC.

The sorting was intended for the `slot` column, like we can see in the original code.
However, currently the ASC does nothing since the sorting is ASC by default anyways - at least on MySQL.

Reply:
lol everybody knows it's ASC/DESC per column, i don't get why you said that the borzis and changed it... no sense

but as rochet2 said, the default is ASC, but we put ASC on item to make it clearer to the reader

revert back to the last commit imo

Reply:
sorry, I thought that was a mistake and not specifically meant

Reply:
restored, sorry xD

Reply:
np np